tools/functional-tester/etcd-tester/tester.go

```diff
@@ -33,6 +33,9 @@ type tester struct {
 
 	stresser Stresser
 	checker  Checker
+
+	runnerMode string
+	runner     Runner
```

Comment:
This shouldn't be special cased; one interface should be enough. Everything should go through stresser/checker. Otherwise the code becomes bifurcated and needs separate reasoning both stressers and checkers. Use an adapter pattern?